lite-pm/styles/AppPage.module.css

```diff
@@ -17,4 +17,19 @@
 .todoHeader {
     font-size: 15pt;
     margin-top: 12pt;
-}
\ No newline at end of file
+}
+
+.duration {
+    width: 30pt;
```

Comment:
Just a note, but for the future, it's more common to use `px` instead of `pt`. They are not interchangeable